src/api/python/CMakeLists.txt

```diff
@@ -19,6 +19,35 @@ if(POLICY CMP0057)
   cmake_policy(SET CMP0057 NEW)
 endif()
 
+# Check that scikit-build is installed
+# Provides CMake files for Python bindings
+check_python_module("skbuild" "scikit-build")
+
+# setuptools for installing the module
+check_python_module("setuptools")
+
+# Find cmake modules distributed with scikit-build
+# They are distributed under <scikit-build directory>/resources/cmake
+execute_process(
+  COMMAND
+    ${PYTHON_EXECUTABLE} -c "from __future__ import print_function;\
+                             import os; import skbuild;\
+                             print(os.path.dirname(skbuild.__file__) + \
+                             os.path.sep + 'resources' + os.path.sep + 'cmake', end='')"
+  OUTPUT_VARIABLE
+    SKBUILD_CMAKE_MODULE_PATH
+  RESULT_VARIABLE
+    RET_SKBUILD_CMAKE_MODULE_PATH
+  )
+
+if (RET_SKBUILD_CMAKE_MODULE_PATH)
+  message(FATAL_ERROR "Failed to find CMake module path from scikit-build")
+endif()
```

Comment:
If this happens what is the user supposed to do?

Reply:
Hmm.. probably complain to the developers haha. Should I add a message about that? This would happen if they change the directory structure.

Reply:
The CMake modules *always* ship with scikit-build, the question is where you can find them?

Reply:
Right now it ships with `scikit-build` and is always in the same place. So this is actually just guarding against future changes.